python/pyspark/sql/connect/session.py

```diff
@@ -267,6 +269,10 @@ def createDataFrame(
                 [ser._create_batch([(c, t) for (_, c), t in zip(data.items(), arrow_types)])]
             )
 
+            if isinstance(schema, StructType):
+                assert arrow_schema is not None
```

Comment:
May dumb question, but just out of curiosity: is there any change that we get `None` for `arrow_schema`??

Seems like `to_arrow_schema` function always return `pyarrow.Field` since it uses `pa.field` to create the schema?

Reply:
It just makes `mypy` happy. 😄 

btw, `to_arrow_schema` returns `pa.Schema`.

Reply:
Oh, yeah I actually meant that `pa.field` returns `pyarrow.Field`. Sorry for the confusion